### DISCUSSION TITLE
next/image medium.com's image placeholder feature

### DISCUSSION THREAD

**Discussion by larbisahli** [vercel/next.js#18579] [Ideas]
Hello guys, 

In medium.com website we can see that it loads images with a 60px version of the original image (900px) and hides it behind a CSS `filter: blur(20px)` while downloading the original image. 

I think we can do the same thing with `next/image`, for example create a 30px or 60px image from the original image size and use it with the CSS filter as a placeholder while waiting for the other big (900+kb) image to load in the background for the browser to cache it and use it for a smooth user experience.

![placeholder](https://user-images.githubusercontent.com/52937392/97772965-09c06400-1b4c-11eb-91e7-f556ce0c090d.png)

**Comment by balazsorban44** [comment]
Another interesting alternative - for a smiliar effect - could be: https://github.com/woltapp/blurhash

<img src=https://raw.githubusercontent.com/woltapp/blurhash/master/Media/HowItWorks1.jpg width=256/>

**Reply by larbisahli** [reply]
Thank you @balazsorban44  I will try it seems very interesting.

**Reply by balazsorban44** [reply]
https://github.com/vercel/next.js/issues/18858

**Comment by lucasfrey** [comment]
Agree with that, some kind of placeholder would be a great addition

**Comment by styfle** [comment]
The blur-up placeholder feature is available in [Next.js 11](https://nextjs.org/blog/next-11) for static imports!

### Usage

```jsx
import Image from 'next/image'
import pic from '../public/mountains.jpg'

<Image src={pic} placeholder="blur" />
```

### Demo

https://image-component.nextjs.gallery
